release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -308,6 +327,20 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 
 For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support].
 
+[id="ocp-4-14-NRO-image-must-gather"]
+==== Collecting data for the NUMA Resources Operator with the `must-gather` image of the operator
+
+In this release, the `must gather` tool has been updated to collect the NUMA-aware scheduling and NUMA Resources Operator data with the NUMA Resources Operator `must-gather` image.
+
+For further information about gathering debugging information relating to the NUMA Resources Operator, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-about-collecting-nro-data_numa-aware[Collecting NUMA-aware pod scheduling data].
```

Comment:
same here concerning the term "pod scheduling", if we can replace it with NUMA Resources Operator data it would be good.